.github/workflows/check_urls.yml

```diff
@@ -0,0 +1,46 @@
+name: Check URLs
+
+on: [push]
```

Comment:
Shall we at least test it once before merging to the main branch? For instance, add pull_request trigger first and then remove it before merge?